generator/internal/sidekick/templatedata.go

```diff
@@ -155,13 +161,27 @@ func newTemplateData(model *api.API, c language.Codec) *TemplateData {
 	// Delay this until the Codec had a chance to compute what packages are
 	// used.
 	data.RequiredPackages = c.RequiredPackages()
+
+	messagesByID := map[string]*Message{}
```

Comment:
Is this different than model.State.MessageByID?

Reply:
Yes, these are `*sidekick.Message`  not the `*api.Message`.